backend/src/util/mod.rs

```diff
@@ -1 +1,146 @@
 //! This package contains structures that are used in many components.
+//!
+//! For a complete list and explanations you can see [here](https://www.sw-ka.de/media/?file=4458listeallergesetzlichausweisungspflichtigenzusatzstoffeundallergenefuerwebsite160218.pdf&download).
+
+use chrono::NaiveDate;
+
+/// This enum lists every possible allergen a meal can have.
+pub enum Allergen {
```

Comment:
Should we spread them also over multiple files?

Reply:
see above